tests/conftest.py

```diff
@@ -70,7 +71,10 @@ def tmp_dir_session(tmpdir_factory):
 
 class VEnv(EnvBuilder):
     def __init__(self, env_dir):
-        super().__init__(with_pip=True)
+        with warnings.catch_warnings():
+            if sys.version_info[:3] == (3, 11, 0):
+                warnings.filterwarnings('ignore', 'check_home argument is deprecated and ignored.', DeprecationWarning)
```

Comment:
Please add a comment pointing to a bug report or something detailing why this is necessary and when it can be removed.

Reply:
It was added in https://github.com/python/cpython/pull/92899. No set ETA on when it will be removed, but IIRC the policy is to have it deprecated for at least 2 versions, I am tempted to remove it then.

Reply:
Thanks. But what I would like to have is a note that says when the warning suppression can be removed. Most likely this will be when 3.11.1 will be released, but having a pointer to the commit where the code triggering the warning in the standard lib has been fixed would be best. I think that would be https://github.com/python/cpython/pull/98743

Reply:
Yes, that's it. Only warns on 3.11.0 exactly, 3.11.1 should be fine.

Okay to squelch this some other way if you prefer, I just copied the solution we used in build, but in build, it was a user-facing warning. Here it's in the tests, so I could use a pytest method of ignoring warnings if you prefer.